### DISCUSSION TITLE
Unable to access GitHub Global Campus page

### DISCUSSION THREAD

**Discussion by aku1310** [community/community#50025] [GitHub Education]
I am a verified student and have the education pack, yet when I open these links, https://education.github.com/globalcampus/octernships and https://education.github.com/globalcampus/student, I see the following:

![image](https://user-images.githubusercontent.com/72210364/225047906-111ef008-c854-499b-b3fe-dbcae63d2672.png)
![image](https://user-images.githubusercontent.com/72210364/225047940-b218ab39-d4a4-4618-84ba-03afcf743d05.png)

Is there anything wrong with my verification status or something? As I am stuck with this page while accessing the Global Campus for a few days now.

Please let me know how this issue can be resolved.

**Comment by samus-aran** [comment]
Hi @aku1310 👋 

Are you still experiencing this issue?

**Reply by aku1310** [reply]
Hi!

Yes, it's the same error as the one in the above screenshots

**Reply by bonzonic** [reply]
Hi samus-aran! I am experiencing the issues above as well! I'm accessing the links from Malaysia.

**Comment by bonzonic** [comment]
Are there any solutions for this?

**Reply by aku1310** [reply]
It solved automatically for me, but now I'm getting that I'm not eligible for Octernships now.

**Reply by bonzonic** [reply]
Same, feels bad man

**Reply by aku1310** [reply]
Yeah, especially since a few of the octernships have an end date of applying this week only
